RiotTests/FakeUtils.swift

```diff
@@ -346,3 +348,134 @@ class FakeKeyVerificationManager: NSObject, MXKeyVerificationManager {
     }
     
 }
+
+class MockPostHog: PostHogProtocol {
+    
+    private var enabled = false
+    
+    func optIn() {
+        enabled = true
+    }
+    
+    func optOut() {
+        enabled = false
+    }
+    
+    func reset() {
+        
+    }
```

Comment:
We don't have swiftlint/format on this project, but probably nicer to collapse these empty methods (or put a comment in them)
```suggestion
    func reset() { }
```